src/Core/Grid/Definition/Factory/CustomerGroupsGridDefinitionFactory.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Grid\Definition\Factory;
+
+use PrestaShop\PrestaShop\Core\Grid\Action\Row\RowActionCollection;
+use PrestaShop\PrestaShop\Core\Grid\Action\Row\Type\LinkRowAction;
+use PrestaShop\PrestaShop\Core\Grid\Column\ColumnCollection;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\ActionColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\BulkActionColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\DataColumn;
+use PrestaShop\PrestaShop\Core\Grid\Filter\Filter;
+use PrestaShop\PrestaShop\Core\Grid\Filter\FilterCollection;
+use PrestaShopBundle\Form\Admin\Type\SearchAndResetType;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\HttpFoundation\Request;
+
+class CustomerGroupsGridDefinitionFactory extends AbstractGridDefinitionFactory
+{
+    public const GRID_ID = 'customer_groups';
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getId()
+    {
+        return self::GRID_ID;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getName()
+    {
+        return $this->trans('Customer Groups', [], 'Admin.Navigation.Menu');
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getColumns()
+    {
+
+        return (new ColumnCollection())
+            ->add(
+                (new BulkActionColumn('title_bulk'))
+                    ->setOptions([
+                        'bulk_field' => 'id_group',
+                    ])
+            )
+            ->add(
+                (new DataColumn('id_group'))
+                    ->setName($this->trans('ID', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'id_group',
+                    ])
+            )
+            ->add(
+                (new DataColumn('name'))
+                    ->setName($this->trans('Name', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'name',
+                    ])
+            )
+            ->add(
+                (new DataColumn('reduction'))
+                    ->setName($this->trans('Reduction', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'reduction',
+                    ])
+            )
+            ->add(
+                (new DataColumn('price_display_method'))
+                    ->setName($this->trans('Price display Method', [], 'Admin.Global'))
```

Comment:
```suggestion
                    ->setName($this->trans('Price display Method', [], 'Admin.Shopparameters.Feature'))
```

Not every wording is part of the `Admin.Global` namespace, when you use a wording you should always check if it exists and if it does which namespace it belongs to. For this one, I did a quick search of `'Price display Method'` and found the proper namespace. You should check the other yourself, as much as possible we try to avoid adding ne wordings when there are already some existing ones.